client/src/index.jsx

```diff
@@ -12,14 +12,21 @@ class App extends React.Component {
     super(props);
     this.state = {
       id: 28212,
-      value: ''
+      value: '',
+      starValue: 0
     };
   }
 
   onChange (e) {
     this.setState({ value: e.target.value });
   }
 
+  setStars (value) {
+    this.setState({
+      starValue: value
```

Comment:
Is this the same value as our this.state.value? Potential watch out for like variable names.